crates/holochain/src/test_utils.rs

```diff
@@ -117,9 +117,6 @@ pub async fn install_app(
     data: &[(DnaFile, Option<MembraneProof>)],
     conductor_handle: ConductorHandle,
 ) {
-    for (dna, _) in data.iter() {
```

Comment:
Again, this is redundant. They are also registered as part of `install_app_minimal()` below.